ONSdigital/design-system
# Add suffix option to lists and fix link icon colour hover

### What is the context of this PR?
Fixes: #1173 
- Updates case of "forum" in external links to be more consistent
- Adds suffix option to list items.
- Updated css so that icons in icon links now change colour on hover

### How to review 
- Check suffix option works for lists
- Check icons in icon links change to correct colour on hover
